source/codegen/metadata/nidigital/functions.py

```diff
@@ -1427,7 +1427,7 @@
         "returns": "ViStatus"
     },
     "GetChannelName": {
-        "codegen_method": "no",
+        "codegen_method": "public",
```

Comment:
Document these changes in Changes.md file

Reply:
I will create separate pr for changes.md and document the changes there